CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Added
 * **BREAKING: dso-toolkit + core + css + react:** Dropdown Menu (Web Component en CSS Component update) ([#1173](https://github.com/dso-toolkit/dso-toolkit/issues/1173))
   * `.dropdown` wordt `.dso-dropdown-menu`, `.dropdown-menu` wordt `.dso-dropdown-options`, en... **Markup changes, see PR ([#1225](https://github.com/dso-toolkit/dso-toolkit/pull/1225))**
+* **dso-toolkit + core + css + react + sources:** Checkbox voorzien van indeterminate state en nestable maken ([#1185](https://github.com/dso-toolkit/dso-toolkit/issues/1185))
```

Comment:
Dit is 'n beetje misleidend, want voor `core` is de Selectable nog niet nestable toch?

Reply:
Ik snap wat je zegt, hij is niet nestable in `core` maar wel indeterminate. Hoe zouden we dit beter kunnen noteren?